RELEASE/scripts/postsool.ash

```diff
@@ -663,9 +664,15 @@ void handlePostAdventure()
 		{
 			doML = false;
 		}
+		if(my_location() == $location[The Smut Orc Logging Camp])
```

Comment:
Honestly, I think all this "do ML" "don't do ML" stuff should definitely be in presool, if I'm not missing something obvious.